cmd/flags/flags.go

```diff
@@ -44,14 +44,26 @@ func AddConfigFileFlag(flagSet *flag.FlagSet) {
 }
 
 // TryLoadConfigFile initializes viper with config file specified as flag
-func TryLoadConfigFile(v *viper.Viper, logger *zap.Logger) {
+func TryLoadConfigFile(v *viper.Viper) error {
 	if file := v.GetString(configFile); file != "" {
 		v.SetConfigFile(file)
 		err := v.ReadInConfig()
 		if err != nil {
-			logger.Fatal("Error loading config file", zap.Error(err), zap.String(configFile, file))
+			errors.Wrap(err, fmt.Sprintf("Error loading config file %s", file))
 		}
 	}
+	return nil
+}
+
+// NewLogger pareses log Level
+func (flags *SharedFlags) NewLogger(conf zap.Config, options ...zap.Option) (*zap.Logger, error) {
```

Comment:
it looks like in all cases you're passing zap.Production() as the config. Is there a use case you're thinking of when it's not going to be production?  Also, do we need a way to affect that config from cmd line / our config?

Reply:
> Is there a use case you're thinking of when it's not going to be production? Also, do we need a way to affect that config from cmd line / our config?

Not right now. Maybe a debug flag or something like that in the future. I made this as flexible as possible. 

So do you want to change it to the following?
```go
func (flags *SharedFlags) NewLogger() (*zap.Logger, error) 
```